[2023-03-29 15:55:46,212] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_merge_dag.merge_task manual__2023-03-29T15:55:44.094028+00:00 [queued]>
[2023-03-29 15:55:46,221] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_merge_dag.merge_task manual__2023-03-29T15:55:44.094028+00:00 [queued]>
[2023-03-29 15:55:46,221] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 15:55:46,221] {taskinstance.py:1357} INFO - Starting attempt 1 of 4
[2023-03-29 15:55:46,221] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 15:55:46,235] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): merge_task> on 2023-03-29 15:55:44.094028+00:00
[2023-03-29 15:55:46,240] {standard_task_runner.py:52} INFO - Started process 173 to run task
[2023-03-29 15:55:46,242] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'data_merge_dag', 'merge_task', 'manual__2023-03-29T15:55:44.094028+00:00', '--job-id', '609', '--raw', '--subdir', 'DAGS_FOLDER/data_merge.py', '--cfg-path', '/tmp/tmp9tb56b_o', '--error-file', '/tmp/tmp3kaiybjf']
[2023-03-29 15:55:46,242] {standard_task_runner.py:80} INFO - Job 609: Subtask merge_task
[2023-03-29 15:55:46,290] {task_command.py:369} INFO - Running <TaskInstance: data_merge_dag.merge_task manual__2023-03-29T15:55:44.094028+00:00 [running]> on host 4513f77b4fd8
[2023-03-29 15:55:46,358] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=kartaca
AIRFLOW_CTX_DAG_ID=data_merge_dag
AIRFLOW_CTX_TASK_ID=merge_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-29T15:55:44.094028+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-29T15:55:44.094028+00:00
[2023-03-29 15:55:46,407] {python.py:173} INFO - Done. Returned value was: None
[2023-03-29 15:55:46,420] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=data_merge_dag, task_id=merge_task, execution_date=20230329T155544, start_date=20230329T155546, end_date=20230329T155546
[2023-03-29 15:55:46,455] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-29 15:55:46,552] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
